release/stable/opensuse423/docker/Dockerfile

```diff
@@ -0,0 +1,88 @@
+# Docker image file that describes an OpenSuse 42.3 image with PowerShell installed from tar.gz
+# Implemented in two stages, one that downloads and uncompresses the tar.gz
+# The next stage simply copies the files from the first stage, 
+# effectively removing the tar.gz and packages that are not needed from the final image.
+
+# Define arg(s) needed for the From statement
+ARG fromTag=42.2
+
+FROM opensuse:${fromTag} AS installer-env
+
+
+# Define Args for the needed to add the package
+ARG PS_VERSION=6.1.0
+ARG PS_PACKAGE=powershell-${PS_VERSION}-linux-x64.tar.gz
+ARG PS_PACKAGE_URL=https://github.com/PowerShell/PowerShell/releases/download/v${PS_VERSION}/${PS_PACKAGE}
+ARG PS_INSTALL_VERSION=6
+
+# define the folder we will be installing PowerShell to
+ENV PS_INSTALL_FOLDER=/opt/microsoft/powershell/$PS_INSTALL_VERSION
+
+# Create the install folder
+RUN mkdir -p ${PS_INSTALL_FOLDER}
+
+# Install dependencies
+RUN zypper --non-interactive update --skip-interactive
+
+RUN zypper --non-interactive install \
+        tar
+
+# downoad the Linux tar.gz and save it
+ADD ${PS_PACKAGE_URL} /tmp/linux.tar.gz
+
+# Unzip the Linux tar.gz
+RUN tar zxf /tmp/linux.tar.gz -C ${PS_INSTALL_FOLDER}
+
+# ------ Second stage ------
+# Start a new stage so we loose all the tar.gz layers from the final image
+FROM opensuse:${fromTag}
+
+# Install dependencies
+RUN zypper --non-interactive update --skip-interactive \
+    && zypper --non-interactive install \
+        glibc-locale \
+        glibc-i18ndata \
+        libunwind \
+        libicu \
+        openssl
+
+# Copy only the files we need from the previous stag
+COPY --from=installer-env ["/opt/microsoft/powershell", "/opt/microsoft/powershell"]
+
+# Define Args and Env needed to create links
+ARG PS_INSTALL_VERSION=6
+ENV PS_INSTALL_FOLDER=/opt/microsoft/powershell/$PS_INSTALL_VERSION \
+    \
+    # Define ENVs for Localization/Globalization
+    DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=false \
+    LC_ALL=en_US.UTF-8 \
+    LANG=en_US.UTF-8
+
+RUN localedef --charmap=UTF-8 --inputfile=en_US $LANG \
+    \
+    # Create the pwsh symbolic link that points to powershell
+    && ln -s ${PS_INSTALL_FOLDER}/pwsh /usr/bin/pwsh
+
+# Define args needed only for the labels
+ARG PS_VERSION=6.1.0
+ARG IMAGE_NAME=mcr.microsoft.com/powershell:opensuse-42.3
+ARG VCS_REF="none"
+# Add label last as it's just metadata and usere a lot a parameters
```

Comment:
typo usere -> uses

uses a lot **of** parameters

Reply:
fixed here and in alpine